main.css

```diff
@@ -0,0 +1,135 @@
+body {
+  padding: 0;
+  margin: 0;
+  font-family:
+    -apple-system,
+    BlinkMacSystemFont,
+    'Segoe UI',
+    Roboto,
+    Oxygen,
+    Ubuntu,
+    Cantarell,
+    'Open Sans',
+    'Helvetica Neue',
+    sans-serif;
+}
+
+#page-maincontainer {
+  display: flex;
```

Comment:
- [ ] I can see that you have repeated the property `display: flex;` multiple times in your CSS file, it is better to use DRY because if the information needs to be changed, there is only one place to change it, thus this will make your code more maintainable.

For example, you can have one class that takes all the properties for the display flex like this:
✅ 
```
.display-flex {
  display: flex;
  justify-conent: center;
  align-items: center;
}
```
Instead of this: ❌ 
```
##page-maincontainer {
  display: flex;
  justify-content: center;
  align-items: center;
}
.form-wrapper {
  display: flex;
  justify-content: center;
}

```
And then you can just copy the class `display-flex` everywhere you need to use it in your HTML like this 
`<div class="[custom class] display-flex"></div>`. 

You can learn more about DRY [here](https://www.stat.auckland.ac.nz/~paul/ItDT/HTML/node23.html).